apis/filesystem/src/main/java/org/jclouds/filesystem/strategy/internal/FilesystemStorageStrategyImpl.java

```diff
@@ -691,6 +698,8 @@ public BlobAccess getBlobAccess(String containerName, String blobName) {
 
    @Override
    public void setBlobAccess(String container, String name, BlobAccess access) {
+      filesystemContainerNameValidator.validate(container);
+      filesystemBlobKeyValidator.validate(name);
```

Comment:
Should this also apply to `getBlobAccess`, `putBlob`, and `removeBlob`?  `getBlobKeysInsideContainer` too I guess.

Reply:
These methods have been applied by the validators. I just added some missing ones.